lib/src/window_manager.dart

```diff
@@ -177,11 +177,45 @@ class WindowManager {
 
   /// Returns `bool` - Whether app is active
   ///
-  /// @platforms macos
+  /// @platforms macos,windows
   Future<bool> isActive() async {
     return await _channel.invokeMethod('isActive');
   }
 
+  /// Returns `bool` - Whether app is hidden on Blur Event
+  ///
+  /// @platforms windows
+  Future<bool> isHiddenOnBlur() async {
+    return await _channel.invokeMethod('isHiddenOnBlur');
+  }
+
+  /// Sets whether the window is hidden on Blur event.
+  ///
```

Comment:
Would it be possible to add a comment explaining what exactly a blur event is? Could be by pointing to docs as well.

Reply:
@ksm-android

Reply:
@bselwe can [this](https://github.com/joinbuzz/window_manager/pull/2#discussion_r1048471352) be helpful?